srcpkgs/python3-slugify/template

```diff
@@ -1,21 +1,19 @@
 # Template file for 'python3-slugify'
 pkgname=python3-slugify
-# Please don't update. Only required by Transifx-client,
-# doesn't work with newer versions
-version=1.2.6
-revision=4
+version=6.1.2
+revision=1
 wrksrc="python-slugify-${version}"
 build_style=python3-module
-pycompile_module="slugify"
 hostmakedepends="python3-setuptools"
-makedepends="python3-text-unidecode"
 depends="python3-text-unidecode python3-Unidecode"
 short_desc="Python Slugify application that handles Unicode"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="MIT"
 homepage="https://github.com/un33k/python-slugify"
-distfiles="https://github.com/un33k/python-slugify/archive/${version}.tar.gz"
-checksum=a360fd694bf4b78dedb224324e6c3b05682ff1b0354a95b5f78ba1108f3388f8
+distfiles="${PYPI_SITE}/p/python-slugify/python-slugify-${version}.tar.gz"
+checksum=272d106cb31ab99b3496ba085e3fea0e9e76dcde967b5e9992500d1f785ce4e1
+# no tests in package
```

Comment:
does it error because of this? if not, this shouldn't be needed

Reply:
it does, we might have a bug in detecting what python tests to run, or if there are any